auth0/v3/management/users.py

```diff
@@ -126,6 +126,45 @@ def update(self, id, body):
         """
         return self.client.patch(self._url(id), data=body)
 
+    def get_roles(self, id):
```

Comment:
Let's rename to `list_roles` to match the rest of the entities. Let's also offer the pagination properties by default as in the `list` method

```suggestion
    def get_roles(self, id, page=0, per_page=25, include_totals=True):
```


You can copy the method docs from there as well https://github.com/auth0/auth0-python/blob/d097f0d237da25ac57191f3c12e43b6bef189ba6/auth0/v3/management/users.py#L33-L56